initgeneration/init.sh.tmpl

```diff
@@ -104,9 +104,8 @@ isCloudNativeFullStack true" >> ${1}
 createConfigurationFiles() {
 	echo -n "${INSTALLPATH}/agent/lib64/liboneagentproc.so" >> "${share_dir}/ld.so.preload"
 
-	set +u # allow variable to be empty
+	# Script fails if empty
 	host_tenant="${im_nodes[${K8S_NODE_NAME}]}"
```

Comment:
Would explicitly check if node name is not in array.
https://stackoverflow.com/questions/3685970/check-if-a-bash-array-contains-a-value

And return the desired error code directly (`exit <errorcode>`) rather than letting bash decide what to do